Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
+      activemodel (= 6.1.3)
+      activesupport (= 6.1.3)
+    activesupport (6.1.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-      zeitwerk (~> 2.2, >= 2.2.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
     builder (3.2.4)
-    concurrent-ruby (1.1.7)
+    concurrent-ruby (1.1.8)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.9.0)
-    i18n (1.8.5)
+    erubi (1.10.0)
+    i18n (1.8.9)
       concurrent-ruby (~> 1.0)
-    json (2.3.1)
+    json (2.5.1)
```

Comment:
### [json](http://flori.github.io/json/)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.1 → 2.5.1

[_[change-log](https://github.com/flori/json/blob/master/CHANGES.md), [source-code](https://github.com/flori/json)_]





<details>
<summary>Commits</summary>

A change of **82** commits. See the full changes on [the compare page](https://github.com/flori/json/compare/v2.3.1...v2.5.1).

These are the first 10 commits:
- (4f471bf) [Fixed unexpected illegal/malformed utf-8 error](https://github.com/flori/json/commit/4f471bf590a6193681ef10fc15da33eed91fc33a)
- (70ae672) [[CI] setup supported JRubies to test against](https://github.com/flori/json/commit/70ae672cfe34eee5ffa1ed84ffced4bf2400b9b2)
- (5b4f1ba) [Fix pure parser with unclosed arrays / objects [Fix #314]](https://github.com/flori/json/commit/5b4f1baef055d2a7be58cdfe7329b1b2f5b748c9)
- (20d7be6) [Move options from #generate and #parse to common area](https://github.com/flori/json/commit/20d7be605a62198ffec4cf74080553c128d404e2)
- (380699d) [Merge pull request #430 from BurdetteLamar/master](https://github.com/flori/json/commit/380699de48bf697e9c9610858d4c897ebb4bd79f)
- (6076780) [Do not add a relative path to $LOAD_PATH](https://github.com/flori/json/commit/607678068648e2922c3728b4e5e954ff5bb4a9ef)
- (b6e6a4c) [Merge pull request #431 from mame/avoid-relative-path-in-loa…](https://github.com/flori/json/commit/b6e6a4c894c2dce2f92ab2f9752162189752d798)
- (438d85e) [test/json/json_fixtures_test.rb: Prevent an "out of range" w…](https://github.com/flori/json/commit/438d85e0407902cf76c497662be5381ea7f9d4c5)
- (6174286) [Merge pull request #432 from mame/prevent-warning-during-tes…](https://github.com/flori/json/commit/617428650d48db8a4b5439968e441f859605c6f3)
- (33e64ef) [Enhance RDoc for JSON.parse](https://github.com/flori/json/commit/33e64ef255948923c26ef4f6eb9c2909f30e7b46)


</details>